src/json.hpp

```diff
@@ -9017,38 +9039,41 @@ class basic_json
 
     Serialization function for JSON values. The function tries to mimic
     Python's `json.dumps()` function, and currently supports its @a indent
-    parameter.
+    and @a ensure_ascii parameters.
 
     @param[in] indent If indent is nonnegative, then array elements and object
     members will be pretty-printed with that indent level. An indent level of
     `0` will only insert newlines. `-1` (the default) selects the most compact
     representation.
     @param[in] indent_char The character to use for indentation if @a indent is
     greater than `0`. The default is ` ` (space).
+    @param[in] ensure_ascii If ensure_ascii is true, all non-ASCII characters
+    in the output are escaped with \uXXXX sequences, and the result consists
+    of ASCII characters only.
 
     @return string containing the serialization of the JSON value
 
     @complexity Linear.
 
     @liveexample{The following example shows the effect of different @a indent
-    parameters to the result of the serialization.,dump}
```

Comment:
The `,` here is required. It separates the description (`The following example...`) from the name of the example `dump` (see file [`doc/examples/dump.cpp`](https://github.com/nlohmann/json/blob/develop/doc/examples/dump.cpp)).

Please undo this change.